src/middleware/devtools.ts

```diff
@@ -1,5 +1,39 @@
 import { GetState, PartialState, SetState, State, StoreApi } from '../vanilla'
 
+type DevtoolsOptions = {
```

Comment:
If you're interested you can look up the [repo](https://github.com/reduxjs/redux-devtools), find out what other options we're missing that don't require implementation changes and add jsdocs for the them and the existing ones too